ui/src/app/views/account-creation-success/component.ts

```diff
@@ -16,7 +16,11 @@ export class AccountCreationSuccessComponent {
       private app: AppComponent,
       private account: AccountCreationComponent,
       private signInService: SignInService) {
-    app.backgroundImgSrc = '/assets/images/congrats-female@2x.jpg';
+    // This is a workaround for ExpressionChangedAfterItHasBeenCheckedError from angular
+    setTimeout(() => {
```

Comment:
It would need to go into a helper function thing outside of the components. I can definitely work on it. I wonder if thats a necessary/helpful abstraction though, since it is another thing for new devs to learn, and obscures what it's actually doing. Thoughts?

Reply:
`runNextTick` is kind of nice, but I'd still rather not make a function out of this b/c there are several ways around the same issue - you can inject the change detector and explicitly re-run change detection, you can run things outside Angular's zone, etc - and also b/c sometimes this error _is_ actually a bug waiting to happen.

The error only pops up in the first place when something changes "in between" change detection rounds; we use `setTimeout` to force the change to occur next tick - which may not always be what we want when we see that error, so I'd rather not codify it as a function.